src/util/hass-util.html

```diff
@@ -47,7 +47,7 @@
 
 window.hassUtil.relativeTime = function (dateObj) {
   var delta = Math.abs(new Date() - dateObj) / 1000;
-  var format = new Date() > dateObj ? '%s ago' : 'in %s';
+  var format = new Date() >= dateObj ? '%s ago' : 'in %s';
```

Comment:
This won't solve it, as there is going to be a ms diff.
Instead look at delta-without-abs sign.